docs/blog/20230701-Daeraxa-JulyUpdate.md

```diff
@@ -0,0 +1,127 @@
+
+This wasn't anything to do with our code or builds but was instead a failure of the CI machine used to produce those builds failing to properly connect to various services.
+
+This has now been resolved so if you were waiting for an ARM binary then these are fully available on our [downloads](https://pulsar-edit.dev/download.html) page as normal and for those relying on a community package or distribution then any blocker will now have been cleared.
+
+## Subreddit closure/reopening and possible Lemmy community
+
+<img src="./assets/reddit-private.png" height=200><img src="./assets/lemmy-icon.png" height=200>[^1]
+
+You may or may not be aware that we decided to [close our subreddit](https://www.reddit.com/r/pulsaredit/comments/146ja5a/rpulsaredit_is_taking_part_in_the_reddit_blackout/) as part of the protest against the Reddit API changes and this remained closed for a period of time after the original 48 hours for various reasons.
+
+We recently decided to [re-open the subreddit](https://www.reddit.com/r/pulsaredit/comments/14ktr3q/rpulsaredit_is_reopening_read_on_for_our/) for a number of reasons, some because we don't want to abandon a portion of our community entirely and partly because of threats from Reddit themselves.
+
+The reason I bring this up here is because it is already possible that some people who were using Reddit for Pulsar news have now left it and won't have seen the above post. This post also contains a poll to get feedback from the affected community to see what our next steps would be. Should we close the subreddit and abandon it? Should we open a Lemmy community instead? Should we do both?
+
+So if you are interested in either using Reddit or Lemmy then please have a look at the above post and vote in the poll.
+
+## Antivirus `es5-ext` issue
+
+<img src="./assets/malware.png" height=200>
+
+Some users have been reporting that various antivirus applications and virus scanning tools have been identifying Pulsar as a virus due to the `es5-ext` package.
+
+This is is nothing to worry about, Pulsar is not a virus nor do we use dependencies that are.
+
+After some investigation we discovered that somewhere in our dependency tree (i.e. a package dependency of a package we use) exists this `es5-ext` package. At some point this package was updated to display a "message of peace" to Russians (in reference to the ongoing conflict). This message displays to anyone within the Russian timezones and, whilst the "payload" of a simple message is harmless, the act of targeting based on time-zone has caused this package to be flagged by various scanners and the package author does not seem to want to remove it.
+
+So we have been working on a way to get around this and [@confused-techie] has managed to [pin this to an earlier version](https://github.com/pulsar-edit/pulsar/pull/608) using our own fork of this package.
```

Comment:
```suggestion
So we have been working on a way to get around this and [@confused-techie] with the help of [@savetheclocktower] has managed to [pin this to an earlier version](https://github.com/pulsar-edit/pulsar/pull/608) using our own fork of this package.
```

Also not sure about wording, but while originally I tried to pin to an older unaffected version, or fork is the latest of this package, but removes the offending code. So maybe "has managed to fork the package and remove the offending code that was being flagged as malicious"